index.js

```diff
@@ -70,6 +70,7 @@ async function main() {
 
 		// create right patch document depending on the action tied to the issue
 		// update the work item
+		console.log(`Action: ${vm.action}`)
```

Comment:
do you mean for this debug code to remain?

Reply:
Yes. I want to make sure that we are seeing all actions coming in and that we have a case for them.